### ISSUE TITLE
problem with 10.12 with CoreDisplay patch on mac mini 2012

### ISSUE THREAD

**Issue by mcphone2004** [Floris497/mac-pixel-clock-patch-V2#87]
I used to be able to get 4K @24Hz on SE42UM with older version of OS X + switchresx + pixel patch. 
After upgrade to 10.12 and use the patch, it does not work anymore.
At the beginning, I thought that may have something to do with my custom resolution settings in switchresx.

I used my friend MacBook pro 2015 which also has switchresx to connect to SE42UM in 4K@24/25/30Hz and copy out custom resolution settings and apply it on mac mini 2012 but still gives me a black screen. I think I can rule out that the custom setting is bad. Any suggestion?

I wonder what other people did to revert back from black screen from a mac mini?  In my case, I use an iOS device to access to the screen through duet to change back to the original resolution. Sometimes it didn't work and I need to reinstall the OS. Want to see if there is a better step to recover from such a state. Thanks.

**Comment by Floris497** [comment]
Yes.. you can undo the patch from recovery. `mv` the backup over the patched file `mv <backup file> <patched file>` for the rest..

**Comment by sn00zerman** [comment]
I had exactly the same issue, and solved it by disabling the SwitchResX daemon.
Next, reboot you mac, and go to the default screen settings of your mac, you will see that the custom resolutions like 3840x2160 are available to choose from. (Please, first adjust your Refresh rate, in case you need a lower one)
I'm currently running my mac mini late 2012 with Sierra 10.12.3 and my samsung 4K Curved display at 3840x2160 @30Hz (And the same trick did also work on my early 2013 Macbook Pro Retina.

best regards,
Kris
http://www.digitalplayground.be - where fun meets technology ...
